Book-A-Desk.Domain/Reservation.Validator.fs

```diff
@@ -51,7 +55,7 @@ module BookADeskReservationValidator =
     }
             
     let validateCommand (offices: Office list) (cmd : BookADesk) reservationAggregate = result {
-        do! validateEmailIsNotEmpty cmd.EmailAddress
```

Comment:
Should we keep validating that email is not empty? I don't know if your validation shows an error when email is empty.

Reply:
This implementation validates empty email (you will see that in the test cases of InvalidEmails).